pipeline-scripts/buildlib.groovy

```diff
@@ -23,6 +24,7 @@ def initialize(test=false, regAws=false) {
 
     GITHUB_URLS = [:]
     GITHUB_BASE_PATHS = [:]
+    GITHUB_BASE = "git@github.com:openshift"
```

Comment:
Why is this needed? I saw it was already initialized at declaration.

Reply:
yeah thought that too, then I saw `GITHUB_URLS` and `GITHUB_BASE_PATHS` initialized twice as well, and my scarce Groovy knowledge made me worry. Shall I remove all those duplicated initializations?